pages/content/amp-dev/documentation/guides-and-tutorials/integrate/embed-stories.md

```diff
@@ -0,0 +1,189 @@
+      </p>
+        <amp-story-player style="width: 360px; height: 600px;">
+          <a
+          href="https://preview.amp.dev/documentation/examples/introduction/stories_in_amp/"
+          style="--story-player-poster: url('https://amp.dev/static/samples/img/story_dog2_portrait.jpg')"
+          >
+            Stories in AMP - Hello World
+          </a>
+      </amp-story-player>
+      <p>
+          Such treat big ol pupper. Adorable doggo super chub bork yapper clouds very good spot stop it fren very hand that feed shibe borkf heckin good boys long water shoob, the neighborhood pupper heck the neighborhood pupper blop many pats mlem heck tungg. noodle horse. Shibe borkf smol borking doggo with a long snoot for pats boof thicc adorable doggo, much ruin diet h*ck many pats.
+      </p>
+    </body>
+</html>
+```
+[/example]
+
+## Embed AMP story player
+
+Displaying an AMP story in a non-AMP page requires the use of the [`amp-story-player`](https://github.com/ampproject/amphtml/blob/master/spec/amp-story-player.md) element. 
+
+### Import scripts
+
+Include the two required scripts in the head of your document:
+
+```html
+<script async src="https://cdn.ampproject.org/amp-story-player-v0.js">
+<link href="https://cdn.ampproject.org/amp-story-player-v0.css" rel='stylesheet' type='text/css'>
+```
+
+The first script imports the logic for the player and the second one sets the default styling.  
+
+### Specify the story
+
+Include the `<amp-story-player>` element inside the document `body`. Then specify the desired story by placing an `<a>` tag inside of the `<amp-story-player>` element. Point the `href` to the story’s location. The `href` may point to the URL of a hosted story or a relative path. Place the title of the story within the `<a>` tags.
+
+```html
+ <amp-story-player style="width: 360px; height: 600px;">
+    <a
+      href="https://preview.amp.dev/documentation/examples/introduction/stories_in_amp/">
+      Stories in AMP - Hello World
+    </a>
+  </amp-story-player>
+```
+
+### Size the player
+
+You may define the story player's `width`, `height`, and other styles inline or as you would any other element's style.
+
+```html
+<body>
+...
+  <amp-story-player style="width: 360px; height: 600px;">
+...
+  </amp-story-player>
+...
+</body>
+```
+
+We recommend maintaining a 3:5 aspect ratio for the best user experience, but you may define any width and height. 
+
+#### Responsive sizing
+
+The story player's responsiveness works as any other block element. Use CSS to maintain width and height ratios, such as the example below:
+
+```css
+amp-story-player {
+  width: 50vw;
+  height: 83.35vw;
+ }
+```
```

Comment:
@sebastianbenz wdyt?

Reply:
Can we move this to inline styles on the `amp-story-player` element?  Our plan is to create embed codes from the amp-story extension that include `width` and `height` inline styles by default, so that will (ideally) be how people are used to seeing the styles written

Reply:
done